### DISCUSSION TITLE
Error Problem avast detecting powershell.exe as a malware a start a five server("Live Server (Five Server)")

### DISCUSSION THREAD

**Discussion by Lilicogamer13** [yandeu/five-server#129] [Help / Questions]
error is reported here: https://stackoverflow.com/questions/78214602/visual-studio-code-error-problem-avast-detecting-powershell-exe-as-a-malware-a-s

**Comment by yandeu** [comment]
Same as https://github.com/yandeu/five-server/issues/110
